app/graphql/mutations/customer_portal/generate_url.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Mutations
+  module CustomerPortal
+    class GenerateUrl < BaseMutation
+      include AuthenticableApiUser
+      include RequiredOrganization
+
+      graphql_name 'GenerateCustomerPortalUrl'
+      description 'Generate customer portal URL'
+
+      argument :id, ID, required: true
+
+      field :url, String, null: false
+
+      def resolve(**args)
```

Comment:
Since there is only one input argument you could simply have: `def resolve(id:)`

Reply:
✅